content/articles/setting-up-hotspot-on-kali-linux/index.md

```diff
@@ -0,0 +1,149 @@
+- [Setting up configuration files](#setting-up-configuration-files)
+- [Using GUI](#using-gui)
+- [Conclusion](#conclusion)
+
+### Introduction
+
+In general a general view, it is a nice idea to disable services that you do not use. Kali makes it easy to do this since network services are disabled by default.
+As long as services remain disabled, they do not pose any security threat. However, you should be careful when you enable them because:
+- It has got no firewall by default, so if they listen on all network interfaces, they are effectively publicly available, that way making it vulnerable.
+
+Kali Linux distro comes also with mobile hotspot ability disabled, creating a mobile hotspot can only be done by customizing the network configuration.
+In this article, we will look into the processes that we will use to create a mobile  hotspot on Kali Linux and other Linux distros like Ubuntu. Throughout this tutorial, we are going to be using Kali Linux to create our hotspot. A person using any other distribution can follow through.
+
+### Prerequisites
+
+You have to be familiar with the Linux family of operating systems.
+Introduction to Kali Linux basics and the way to use it can be gotten in the link below.
+[Getting started with kali linux](/engineering-education/getting-started-with-kali-linux/)
+
+### Finding wireless drivers
+
+To access the available network drivers, type the following command in the terminal: 
+```bash
+lspci
+```
+The above command works only for wireless drivers.
+In the case of using a USB adapter use the command shown below:
+ ```bash
+ lsusb
+ ```
+ The output is as follows:
+ ![Drivers](/engineering-education/setting-up-hotspot-on-kali-linux/drivers.png)
+### Installation of the necessary tools
+In order to create our hotspot, we need to install the required tools. The first and crucial tool is **hostapd** which acts as the hotspot server.
+To install **`hostapd`**, type in the following command:
+
+```bash
+ apt-get install hostapd
+ ```
+The second tool will be **dnsmasq** which will act like the server's **dhcp**.
+ To install **`dnsmasq`**, type in the following:
+ 
+```bash
+ apt-get install dnsmaq
+ ```
+Once we're done with the installations, we can now move to the next step.
+We shall proceed to check for the just installed processes. If already started, stop those in progress, and deter them from starting during start-up of the system.
+
+Below are the commands used for stopping and disabling updates of the services:
+
+```bash
+sudo service hostapd stop
+```
+```bash
+sudo service dnsmasq stop
+```
+```bash
+sudo update-rc.d hostapd disable
+```
+```bash
+sudo update-rc.d dnsmasq disable
+```
+The output is as follows:
+
+![Installations](/engineering-education/setting-up-hotspot-on-kali-linux/installations.png)
+
+ ###  Setting up configuration files
+After installing the services, we shall then need to configure the necessary files i.e **`hotspot.sh`**.
+
```

Comment:
Why do we need to configure that file? This is an examples of details that your tutorial is missing. Check the whole tutorial and add information where necessary.